boxoffice/messageclient.py

```diff
@@ -0,0 +1,18 @@
+import requests
+
+from baseframe import _
+
+from . import app, rq
+
+
+@rq.job('boxoffice')
+def send_telegram_message(buyer_fullname, line_item_title):
+    with app.test_request_context():
+        message_text = _(f'{buyer_fullname} purchased {line_item_title}')
```

Comment:
`f` strings can't be used inside `_()` as it will send the rendered string for translation. Change to:

```python
message_text = _("{user} purchased {title}").format(user=buyer_fullname, title=line_item_title)
```